components/scg/src/scg-layout.js

```diff
@@ -55,7 +55,7 @@ SCg.LayoutAlgorithmForceBased.prototype.start = function () {
         .nodes(this.nodes)
         .links(this.edges)
         .size(this.rect)
-        .friction(0.9)
+        .friction(0.5)
```

Comment:
It's a moment to discuss: looks like it's good for large structures but for small it's get too minimal distance